microhapdb/nomenclature.py

```diff
@@ -109,7 +109,7 @@ def validate_lab(self):
         lab = self._raw[4:].split("-")[0]
         if not lab.isupper():
             self._validation_warnings.append(ValidationWarnings.LABCASE)
-        if len(lab) < 2 or len(lab) > 4:
+        if len(lab) < 2 or len(lab) > 6:
```

Comment:
The corresponding author requested the label "SCUZJ" for these markers. The Kidd 2016 nomenclature stipulates 2-4 characters for the lab ID: I think 4 is a reasonable limit, but I also think the author's request is reasonable and I don't think there's a reason for strict enforcement at the moment. I relaxed the hard limit to 6, while maintaining the rhetorical limit at 4.